bugswarm/analyzer/result_comparer.py

```diff
@@ -22,8 +22,8 @@ def compare_attributes(reproduced, original, ignore_status=False):
         mismatched_attributes = []
         for attr in original:
             if attr == 'tr_log_tests_failed':
-                reproduced_failed_tests_set = set(reproduced[attr])
-                original_failed_tests_set = set(original[attr])
+                reproduced_failed_tests_set = set(reproduced[attr].split('#'))
```

Comment:
@Robert-Furth Where in the Analyzer does it add the `#` as the delimiter?

Reply:
`log_file_analyzer.py`, lines 288-292, in the `output` function:
```python
attr = getattr(self, mapping[key])
if type(attr) is list:
    output[key] = '#'.join(attr)
else:
    output[key] = attr
```